sqe-dto/User.cs

```diff
@@ -156,5 +156,13 @@ public class EditorDTO
         public string organization { get; set; }
     }
 
+    public class UserDataStoreDTO
+    {
+        [Required]
+        [StringLength(1000000, MinimumLength = 2,
```

Comment:
That is more than enough, I do not expect the data to be more than a few KBs. If we want something more substantial, we will probably add a new database entity.